File: DropZone/DropZone.Web/src/models/aircrafts/aircraft.model.ts
import { DropZoneModel } from "../dropzones/dropzone.model";

export class AircraftModel {
	public Id: number;
	public Type: string;
	public Capacity: number;
	public IsAvailable: boolean;
	public ModifiedAt: Date;
	
	public DropZoneId: number;
	
	public DropZone: DropZoneModel;
};

File: DropZone/DropZone.Core/Models/Aircraft/AircraftModel.cs
using System;

namespace DropZone.Core.Models
{
    public class AircraftModel
    {
        public long Id { get; set; }

        public string Type { get; set; }

        public int Capacity { get; set; }

        public bool IsAvailable { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long DropZoneId { get; set; }


        public DropZoneModel DropZone { get; set; }
    }
}
